src/authx/auth.py

```diff
@@ -241,13 +249,14 @@ def is_action_allowed_for_program(token, method=None, path=None, program=None, o
 
 
 def is_user_candig_authorized(request, token=None):
-    # if the user is in opa, they are CanDIG-authorized
+    # check to see if user_is_candig_authorized is true in the opa permissions
     try:
-        response, status_code = get_self_in_opa(get_auth_token(request, token=token))
+        user_token = get_auth_token(request, token=token)
```

Comment:
I think user_token is never used? Can we remove it?

Reply:
sure